### DISCUSSION TITLE
Tag pill looks weird in Preview Mode

### DISCUSSION THREAD

**Discussion by vizance** [BryanJenksCommunity/FAQ#53] [Q&A] [answered]
Hello Bryan

I just used your Custom CSS today and found the tag pill looks weird in Preview Mode. Please refer to below pictures.

# Issue 1
## In Editor mode
![image](https://user-images.githubusercontent.com/29624657/113746664-25521b00-9739-11eb-9a92-88d3124aeb0d.png)

## In Preview mode
![image](https://user-images.githubusercontent.com/29624657/113746684-2daa5600-9739-11eb-89fd-6bdd37d4975d.png)

The word tag pill looks normal but the emoji tag pill doesn't. 

# Issue 2

This is another issue if I use emoji in the note.

## In Editor mode
![image](https://user-images.githubusercontent.com/29624657/113750768-d0fd6a00-973d-11eb-9a48-4c2c265c930e.png)

## In Preview mode
![image](https://user-images.githubusercontent.com/29624657/113750825-d78be180-973d-11eb-8c9a-e8d1dda4253d.png)

---

I just look up the CSS but don't have any idea. How could I solve it ?

Thanks in advance.

**Comment by tallguyjenks** [comment, marked as answer]
Send me an email and I'll get you an updated copy of the css, I turned tag pills off because of this 

Bryan@bryanjenks.dev

**Reply by vizance** [reply]
Email Sent, thanks

**Comment by vizance** [comment]
Hi Bryan,

Thanks for your file, however I still get the same problems. (actually this file is what I get from subscribing your Newsletter)

What I do now is adding the `Bryans Gruvbox.css` file into my .obsidian/themes folder, not sure I should put other files (ex. preambly.sty) or not.

Should I adjust any obsidian setting ? Thanks (my obsidian version is v0.11.13)

**Reply by vizance** [reply]
2021.04.10 update

I finally find the solution by myself. Refer to [Emojis in links not behaving when using a particular theme - Resolved help - Obsidian Forum](https://forum.obsidian.md/t/emojis-in-links-not-behaving-when-using-a-particular-theme/12337) , anyone who has this issue can add below css after img css setting.

**This is img css in Bryans Gruvbox.css**
```
img
{
    display: block !important;
    margin-left: auto !important;
    margin-right: auto !important;
}
```

**Add below css after**
```
.emoji {
  display: inline-block !important;
}
```

That's it ! Now issues in this thread are all solved.
